segmentation_models_pytorch/encoders/__init__.py

```diff
@@ -51,6 +52,10 @@
 def get_encoder(name, in_channels=3, depth=5, weights=None, output_stride=32, **kwargs):
     if name.startswith("tu-"):
         name = name[3:]
+
+        if "encoder_indices" in kwargs and kwargs["encoder_indices"] is None:
+            kwargs["encoder_indices"] = "first"
```

Comment:
```suggestion
        if "out_indices" in kwargs and kwargs["out_indices"] is None:
            kwargs["out_indices"] = "first"
```
While it's called `encoder_indices` in the function `select_feature_indices`, in `TimmUniversalEncoder` it is still called `out_indices`